main.tf

```diff
@@ -343,8 +363,13 @@ data "autocloud_blueprint_config" "bucket" {
   # Set the KMS Keys
   variable {
     name  = "bucket.variables.encryption_key_names"
-    type  = "raw"
-    value = "{ \"{{namespace}}-{{environment}}-{{name}}\" = {{keyring_name}}[\"default\"] }"
+    //type  = "raw"
+    value = <<HCL
+    {
+      #{{namespace}}-{{environment}}-{{name}} = {{keyring_name}}["default"]
```

Comment:
@ckoning this is something new that was out of our radar, using interpolation for a key in an object or map, I did a jira ticket